.teamcity/_self/projects/WebApp.kt

```diff
@@ -598,8 +598,8 @@ fun playwrightPrBuildType( targetDevice: String, buildUuid: String ): BuildType
 					export NODE_ENV="test"
 					export PLAYWRIGHT_BROWSERS_PATH=0
 
-					# Install modules
-					${_self.yarn_install_cmd}
+					# Install deps
```

Comment:
Optional suggestion: do you think it's worth echoing here something about why we are using focus, and that all the peer dependency errors are to be expected?

I just know that if I wasn't on the PR, this could be the kind of thing I see later in the logs and wonder if all those errors are actually problems.